db/migrate/20150221153705_create_submissions.rb

```diff
@@ -1,11 +1,13 @@
 class CreateSubmissions < ActiveRecord::Migration
   def change
     create_table :submissions do |t|
-      t.timestamps null: false
       t.references :user, null: false
       t.string :step, null: false
       t.json :content, null: false, default: {}
-      t.boolean :finalized, null: false, default: false
+      t.boolean :finalized, null: false, default: false, index: true
```

Comment:
Not for me, I mean, with time migrations grow and grow - so I really prefer to have them compact as long as we can - which means up to live beta.